py/load_firmware.py

```diff
@@ -17,17 +17,18 @@
 
 import sys
 import pprint
+import typing
 from time import sleep
 
 from bitbox02 import devices
 from bitbox02 import Bootloader, BitBox02
 
 
-def eprint(*args, **kwargs):
-    print(*args, **kwargs, file=sys.stderr)
+def eprint(msg: str) -> None:
```

Comment:
The purpose of this function is to transparently wrap `print(*objects, sep=' ', end='\n', file=sys.stdout, flush=False)`. So I suggest that we either do `*args: Any, **kwargs: Any` or `print(*objects: object, sep: str = ' ', end: str = '\n', flush: boolean = False) -> None`

Reply:
Done